libs/server/Storage/Session/ObjectStore/SortedSetGeoOps.cs

```diff
@@ -132,11 +132,7 @@ public unsafe GarnetStatus GeoSearchStore<TObjectContext>(ArgSlice key, ArgSlice
             }
             var objectStoreLockableContext = txnManager.ObjectStoreLockableContext;
 
-            var isMemory = false;
-            MemoryHandle ptrHandle = default;
-            var ptr = output.SpanByte.ToPointer();
-            var curr = ptr;
-            var end = curr + output.Length;
+            var writer = new RespMemoryWriter(functionsState.respProtocolVersion, ref output);
```

Comment:
Would be better to use `using` here to ensure dispose on return (instead of Dipose at the end), i see several code paths that return without disposing below. please check other uses of this too.